lls/cache.c

```diff
@@ -489,6 +489,8 @@ lls_cache_scan(struct lls_config *lls_conf, struct lls_cache *cache)
 void
 lls_cache_destroy(struct lls_cache *cache)
 {
+	rte_free(cache->records);
+	cache->records = NULL;
 	rte_hash_free(cache->hash);
```

Comment:
1. The hash table should be the first one to be released since it's the last to be allocated. That is the reverse order of creation.

2. It's nice to add `cache->hash = NULL;` after releasing the hash table.